ccd-definition/CaseField.json

```diff
@@ -162,5 +162,23 @@
     "Label": "Response deadline",
     "FieldType": "DateTime",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "systemGeneratedCaseDocuments",
+    "Label": "System generated Case Documents",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "CaseDocument",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "ccdCaseReference",
```

Comment:
@dharmendrak  / @swalker125 - is there still an outstanding question on why this attribute is being repeated in the data object?

Reply:
removed